UDV-Core/src/Modules/Temporal/views/TemporalGraphWindow.js

```diff
@@ -0,0 +1,80 @@
+import { Window } from "../../../Utils/GUI/js/Window";
+import { NetworkManagerSingleton } from "../viz";
+import './TemporalWindow.css';
+
+// TODO: MANAGE DATES WITH MOMENT
+
+/**
+ * Constructor for TemporalWindow Class
+ * Handles the GUI part enabling the user to specify the chosen "time" (moment or
+ * date) of observation for displayal of the (3D) scene.
+ * Note that it is not the Temporal View's responsability to
+ * alter/modify/update the scene according to the user specified moment (but
+ * only to trigger the possible hook-ups).
+ * This View represents a timestamp with the Moment.js library.
+ * @param refreshCallback : callback to be called when the time has changed.
+ * @param options : optional parameters (min time, max time and current time)
+ */
+export class TemporalGraphWindow extends Window {
```

Comment:
Can you factorize the code that is shared with `TemporalWindow` with class inheritance please ? To this end, please rename `TemporalWindow` to `TemporalSliderWindow` and use the name `TemporalWindow` for the parent class.

Reply:
Added

Reply:
Well the idea was to actually factorize the code, i.e. avoid to duplicate code in `TemporalGraphWindow` and in `TemporalSliderWindow` by putting it into the parent class `TemporalWindow`, not to create an empty class... For instance, the beginning of the constructors are the same.

Reply:
Cleaned